presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/GeometryUtils.java

```diff
@@ -13,15 +13,19 @@
  */
 package com.facebook.presto.geospatial;
 
+import com.esri.core.geometry.Envelope;
 import com.esri.core.geometry.Geometry;
 import com.esri.core.geometry.GeometryCursor;
 import com.esri.core.geometry.SpatialReference;
 import com.esri.core.geometry.ogc.OGCConcreteGeometryCollection;
 import com.esri.core.geometry.ogc.OGCGeometry;
+import com.google.common.base.Verify;
```

Comment:
I think we used to use `checkState` from `Preconditions`. `Verify` is tagged as `@Beta` class.